commit ef71a3e15f63f8cd886b44a3c2acff3c5ed95f45
Author: Friedrich W. H. Kossebau <kossebau@kde.org>
Date:   Tue Feb 4 20:56:36 2014 +0100

    Install missing headers (referenced from others)
    
    Thanks to Maximiliano Curia for hint

diff --git a/gui/CMakeLists.txt b/gui/CMakeLists.txt
index 9c5700a..63e40ac 100644
--- a/gui/CMakeLists.txt
+++ b/gui/CMakeLists.txt
@@ -82,6 +82,10 @@ set( oktetagui_LIB_HDRS
   line.h
   linepositionrange.h
   linerange.h
+  coord.h
+  coordrange.h
+  coordrangelist.h
+  selection.h
   abstractbytearraycolumnrenderer.h
   offsetcolumnrenderer.h
   bordercolumnrenderer.h
@@ -92,7 +96,7 @@ set( oktetagui_LIB_HDRS
   abstractbytearrayview.h
   bytearraycolumnview.h
   bytearrayrowview.h
-  bytearraytablelayout.h 
+  bytearraytablelayout.h
   bytearraytableranges.h
   offsetformat.h
 )
diff --git a/kasten/controllers/CMakeLists.txt b/kasten/controllers/CMakeLists.txt
index 8648b9f..fc7006e 100644
--- a/kasten/controllers/CMakeLists.txt
+++ b/kasten/controllers/CMakeLists.txt
@@ -491,6 +491,7 @@ set( okteta_kastencontrollers_LIB_HDRS
   view/bookmarks/bookmarkstoolfactory.h
   view/bookmarks/bookmarkstoolviewfactory.h
   view/overwritemode/overwritemodecontroller.h
+  view/poddecoder/poddata.h
   view/poddecoder/poddecodertool.h
   view/poddecoder/poddecodertoolview.h
   view/poddecoder/poddecodertoolfactory.h
@@ -525,6 +526,7 @@ set( okteta_kastencontrollers_LIB_HDRS
   view/viewstatus/viewstatuscontroller.h
   view/viewprofiles/viewprofilecontroller.h
   view/viewprofiles/viewprofilesmanagecontroller.h
+  view/stringsextract/containedstring.h
   view/stringsextract/stringsextracttool.h
   view/stringsextract/stringsextracttoolview.h
   view/stringsextract/stringsextracttoolfactory.h
@@ -538,6 +540,7 @@ set( okteta_kastencontrollers_LIB_HDRS
   document/info/documentinfotoolview.h
   document/info/documentinfotool.h
   document/overwriteonly/overwriteonlycontroller.h
+  ${CMAKE_CURRENT_BINARY_DIR}/config-qca2.h
 )
 
 kde4_add_library( ${oktetakastencontrollers_LIB}  SHARED ${OKTETA_KASTEN_CONTROLLERS_SRCS} )
